src/components/WearablePreview/EmoteControls/EmoteControls.styled.ts

```diff
@@ -0,0 +1,111 @@
+import styled from "@emotion/styled"
+import { Button } from "../../Button"
+
+const StyledEmoteControlsContainer = styled("div")({
+  display: "flex",
+  margin: "16px 0",
+  padding: "0 24px",
+  position: "absolute",
+  bottom: 0,
+  width: "100%",
+})
+
+const StyledPlayButton = styled(Button)({
+  width: "56px",
+  height: "44px",
+  padding: 0,
+  minWidth: "unset",
+  marginRight: "16px",
```

Comment:
Do you mind using here the `UI standards`? 
```suggestion
  marginRight: theme.spacing(2),
```

and if there is another margin, padding, etc

Reply:
sure!